core/src/data_sources/data_source.rs

```diff
@@ -68,10 +79,35 @@ pub struct Chunk {
     pub score: Option<f64>,
 }
 
-/// Document is used as a data-strucutre for insertion into the SQL store (no chunks, they are
-/// directly inserted in the vector search db). It is also used as a result from search (only the
-/// retrieved chunks are provided in the result). `hash` covers both the original document id and
-/// text and the document metadata and is used to no-op in case of match.
+/// Document is used as a data-strucutre for insertion into the SQL store (no
+/// chunks, they are directly inserted in the vector search db). It is also used
+/// as a result from search (only the retrieved chunks are provided in the
+/// result). `hash` covers both the original document id and text and the
+/// document metadata and is used to no-op in case of match.
+///
+/// Field details - `parents`
```

Comment:
I beleived it was important to document the new field, but not in `docs` since the field is not pushed to external builders. After thinking 5mn, I though the Document model was the right place; if there's something better, glad to move it

Reply:
Sure that's fine